GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java

```diff
@@ -0,0 +1,40 @@
+package hackkerank.day3;
+
+import hackkerank.day1.IfElse;
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class JavaArrayList {
+
+    public static Logger logJava = Logger.getLogger(IfElse.class);
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+        int lines = in.nextInt();
+
+        List[] set = new ArrayList[lines];
```

Comment:
It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList